frontend/src/routes/LandingPage/LandingPageSkeleton.tsx

```diff
@@ -53,3 +67,7 @@ const Title = styled.h1`
     font-size: 1.3rem;
   `};
 `;
+
+const LegoButtonWrapper = styled.div`
+  margin-top: 3em;
+`;
```

Comment:
I might be dumb, but can't you just use styled(LegoBtutton) directly?

Reply:
I tested it out, and personally prefer using the wrapper as I get more properties of the div element that I would have to override in the LegoButton component - making it less dependent on changes to the LegoButton component.